<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerAnnotationModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerAnnotationModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>403</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Annotation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="text">
      <string>Annotations</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="markupsLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Fiducials have moved to the Markups module.</string>
        </property>
       </widget>
      </item>
       <item>
         <widget class="ctkCollapsibleGroupBox" name="CTKCollapsibleGroupBox_Create">
           <property name="title">
             <string>Create-and-place</string>
           </property>
           <layout class="QGridLayout" name="createLayout">
             <item row="0" column="0" >
               <widget class="QLabel" name="createNodeLabel">
                 <property name="toolTip">
                   <string>Select node types to create and place annotations.</string>
                 </property>
                 <property name="text">
                   <string>Select node type: </string>
                 </property>
               </widget>
             </item>
             <item row="0" column="1">
               <widget class="QPushButton" name="createLineButton">
                 <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                   </sizepolicy>
                 </property>
                 <property name="minimumSize">
                   <size>
                     <width>30</width>
                     <height>0</height>
                   </size>
                 </property>
                 <property name="maximumSize">
                   <size>
                     <width>32</width>
                     <height>32</height>
                   </size>
                 </property>
                 <property name="toolTip">
                   <string>Create-and-place an annotation ruler node.</string>
                 </property>
                 <property name="text">
                   <string/>
                 </property>
                 <property name="icon">
                   <iconset>
                     <normaloff>:/Icons/XSmall/MarkupsLine.png</normaloff>:/Icons/XSmall/MarkupsLine.png
                   </iconset>
                 </property>
               </widget>
             </item>
             <item row="0" column="2">
               <widget class="QPushButton" name="createROIButton">
                 <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                   </sizepolicy>
                 </property>
                 <property name="minimumSize">
                   <size>
                     <width>30</width>
                     <height>0</height>
                   </size>
                 </property>
                 <property name="maximumSize">
                   <size>
                     <width>32</width>
                     <height>32</height>
                   </size>
                 </property>
                 <property name="toolTip"><string>Create-and-place an annotation ROI node.</string>
                 </property>
                 <property name="text">
                   <string/>
                 </property>
                 <property name="icon">
                   <iconset>
                     <normaloff>:/Icons/XSmall/MarkupsROI.png</normaloff>:/Icons/XSmall/MarkupsROI.png
                   </iconset>
                 </property>
               </widget>
             </item>
             <item row="0" column="3">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
           </layout>
         </widget>
       </item>
       <item>
       <widget class="ctkCollapsibleGroupBox" name="CTKCollapsibleGroupBox_3">
        <property name="title">
         <string>Edit</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="3">
          <widget class="QPushButton" name="visibleSelectedButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Toggle the visibility of highlighted Annotations. Hierarchies are ignored.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/Icons/Medium/SlicerVisibleInvisible.png</normaloff>:/Icons/Medium/SlicerVisibleInvisible.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="unselectAllButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Unselect all annotations and hierarchies</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../qSlicerAnnotationModule.qrc">
             <normaloff>:/Icons/AnnotationDeselectAll.png</normaloff>:/Icons/AnnotationDeselectAll.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="lockSelectedButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Lock/ unlock highlighted annotations. Hierarchies are ignored.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/Icons/Medium/SlicerLockUnlock.png</normaloff>:/Icons/Medium/SlicerLockUnlock.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="jumpSlicesButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Reset sliceViews to the first coordinate of the highlighted annotation.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../qSlicerAnnotationModule.qrc">
             <normaloff>:/Icons/AnnotationResetView.png</normaloff>:/Icons/AnnotationResetView.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="addHierarchyButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Create a new hierarchy. If a hierarchy is currently highlighted, the new one will be a sub-hierarchy.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/Icons/Medium/SlicerAddHierarchy.png</normaloff>:/Icons/Medium/SlicerAddHierarchy.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QPushButton" name="generateReport">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Display a report of highlighted annotations</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../qSlicerAnnotationModule.qrc">
             <normaloff>:/Icons/AnnotationReport.png</normaloff>:/Icons/AnnotationReport.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="8">
          <widget class="QPushButton" name="deleteSelectedButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Delete highlighted annotations. Hierarchies have to single selected to get deleted.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../qSlicerAnnotationModule.qrc">
             <normaloff>:/Icons/AnnotationDelete.png</normaloff>:/Icons/AnnotationDelete.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="8">
          <widget class="QPushButton" name="unlockHierarchyButton">
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Set all annotations in active list to be unlocked</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/Icons/Small/SlicerUnlock.png</normaloff>:/Icons/Small/SlicerUnlock.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QPushButton" name="lockHierarchyButton">
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Set all annotations in active list locked</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/Icons/Small/SlicerLock.png</normaloff>:/Icons/Small/SlicerLock.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QPushButton" name="invisibleHierarchyButton">
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Set all annotations in active list invisible</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/Icons/Small/SlicerInvisible.png</normaloff>:/Icons/Small/SlicerInvisible.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QPushButton" name="visibleHierarchyButton">
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Set all annotations in active list visible</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/Icons/Small/SlicerVisible.png</normaloff>:/Icons/Small/SlicerVisible.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="4">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="activeHierarchyLabel">
             <property name="toolTip">
              <string>Click a hierarchy to activate it. If the Main Hierarchy is active, new per-annotation lists will be created under it, otherwise annotations will be added to the active list.</string>
             </property>
             <property name="text">
              <string>Active list: </string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="3" column="1" colspan="8">
          <widget class="qMRMLAnnotationTreeView" name="hierarchyTreeView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::InternalMove</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="indentation">
            <number>10</number>
           </property>
           <property name="uniformRowHeights">
            <bool>true</bool>
           </property>
           <property name="listenNodeModifiedEvent" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="selectAllButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Select all annotations and hierarchies</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../qSlicerAnnotationModule.qrc">
             <normaloff>:/Icons/AnnotationSelectAll.png</normaloff>
             <normalon>:/Icons/AnnotationDeselectAll.png</normalon>:/Icons/AnnotationSelectAll.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkDynamicSpacer" name="CTKSpacer">
     <property name="activeSizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inactiveSizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="active">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkDynamicSpacer" name="DynamicSpacer">
     <property name="activeSizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inactiveSizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="active">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDynamicSpacer</class>
   <extends>QWidget</extends>
   <header>ctkDynamicSpacer.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLAnnotationTreeView</class>
   <extends>QTreeView</extends>
   <header>qMRMLAnnotationTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerAnnotationModule.qrc"/>
  <include location="../../../../Base/QTGUI/Resources/qSlicerBaseQTGUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>CTKCollapsibleButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>CTKSpacer</receiver>
   <slot>setActive(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>571</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CTKCollapsibleGroupBox_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>DynamicSpacer</receiver>
   <slot>setActive(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
